deploy/crds/workspaces.ecd.eclipse.org_devworkspacetemplates_crd.yaml

```diff
@@ -0,0 +1,491 @@
+                        type: string
+                      mountSources:
+                        type: boolean
+                      name:
+                        type: string
+                      volumes:
+                        items:
+                          description: Volume that should be mounted to a component
+                            container
+                          properties:
+                            mountPath:
+                              type: string
+                            name:
+                              description: The volume name. If several components
+                                mount the same volume then they will reuse the volume
+                                and will be able to access to the same files
+                              type: string
+                          required:
+                          - mountPath
+                          - name
+                          type: object
+                        type: array
+                    required:
+                    - image
+                    - name
+                    type: object
+                  custom:
+                    description: Custom component
+                    properties:
+                      componentClass:
+                        type: string
+                      embeddedResource:
+                        type: object
+                      name:
+                        type: string
+                    required:
+                    - componentClass
+                    - embeddedResource
+                    - name
+                    type: object
+                  kubernetes:
+                    description: Kubernetes component
+                    properties:
+                      alias:
+                        type: string
+                      inlined:
+                        description: Reference to the plugin definition
+                        type: string
+                      locationType:
+                        description: Type of Kubernetes-like location
+                        type: string
+                      url:
+                        description: Location in a plugin registry
+                        type: string
+                    type: object
+                  openshift:
+                    description: Openshift component
+                    properties:
+                      alias:
+                        type: string
+                      inlined:
+                        description: Reference to the plugin definition
+                        type: string
+                      locationType:
+                        description: Type of Kubernetes-like location
+                        type: string
+                      url:
+                        description: Location in a plugin registry
+                        type: string
+                    type: object
+                  type:
+                    description: Type of project source
```

Comment:
Type of component source?

Reply:
Yes, you're right, but in fact this is generated from the existing description that was already there in the `DevWorkspaceTemplateSpec` Api Go struct. The only new thing is that this struct is now used in 2 main Custom Resource APIs.

So this change is not really part of the source changes done for this PR.

Do you think we could postpone additional fixes in the overall GO API to a distinct issue ?

Reply:
:+1: